cirq-google/cirq_google/experimental/noise_models/calibration_to_fidelity.py

```diff
@@ -0,0 +1,33 @@
+import cirq
+import cirq_google
+import numpy as np
+from cirq.devices.fidelity import Fidelity
+
+def fidelity_from_calibration(calibration: cirq_google.Calibration):
```

Comment:
Please add docstrings for top-level functions. (Nested or private functions don't strictly need a docstring, but a short descriptive comment might be helpful.)